osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -82,13 +83,14 @@ public ChatOverlay()
                         },
                     },
                 },
-                chatContainer = new Container
+                chatContainer = new OsuContextMenuContainer
                 {
                     Name = @"chat container",
                     Anchor = Anchor.BottomLeft,
                     Origin = Anchor.BottomLeft,
                     RelativeSizeAxes = Axes.Both,
                     Height = DEFAULT_HEIGHT,
+                    Masking = true,
```

Comment:
what's the reason for masking here?

Reply:
To avoid this happening ![image](https://i.imgur.com/NNMVWMY.jpg)
Ideally scrolling should be disabled while the context menu is open (in my opinion), but that's a different issue.

Reply:
The context menus should scroll with the scrolling content (they do this elsewhere and this is the look and feel we are going for).